content/updates/2021-01/1-Accelerate-your-connections-with-the-events-api.md

```diff
@@ -0,0 +1,24 @@
+---
+pim_announcement_img: ./img/event-connection-activation.png
+pim_announcement_alt_img: "Events activation in the Connections"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Accelerate your connections with the Events API
+::: meta-data type="New" features="Connectivity" available="mid-January" link-to-doc="https://api.akeneo.com/events-documentation/introduction.html"
+
+You may know how crucial it is to efficiently connect your PIM to third-party applications such as your ERP, eCommerce platform, or DAM. To help you speed up on this topic, we're launching a brand new API, called the `Events API`.
+
+Coupled with our [existing REST API](https://api.akeneo.com/documentation/introduction.html), the Events API offers an additional and complementary way to synchronize PIM data with third-party applications. By reducing synchronization delays, it gives you the power to accelerate the access and distribution of your product information.
```

Comment:
```suggestion
Together with our [existing REST API](https://api.akeneo.com/documentation/introduction.html), the Events API offers an additional and complementary way to synchronize PIM data with third-party applications. By reducing synchronization delays, it gives you the power to accelerate the access and distribution of your product information.
```

Reply:
I wanted to say "Couplée avec notre API REST,". How would you say that? The idea here is to say that the Events API won't replace the REST API, they are complementary and should be used together at the same time.